lib/ddtrace/span.rb

```diff
@@ -40,9 +40,11 @@ def initialize(tracer, name, options = {})
       @trace_id = options.fetch(:trace_id, @span_id)
 
       @meta = {}
+      @metrics = {}
```

Comment:
That was missing so good to have it!

In that case you should update all our tests so that we have both unittests and integration tests with a real agent. We should check also what happens if we put strings, `nil` values, integers, etc...